EIPS/eip-2535.md

```diff
@@ -3,29 +3,15 @@ eip: 2535
 title: Diamonds, Multi-Facet Proxy
 author: Nick Mudge (@mudgen)
 discussions-to: https://github.com/ethereum/EIPs/issues/2535
-status: Last Call
-last-call-deadline: 2020-08-15
+status: Review
+description: Standard for creating modular smart contract systems that can be extended after deployment
 type: Standards Track
 category: ERC
 created: 2020-02-22
 ---
 
 <img align="right" src="../assets/eip-2535/diamond.svg" width="230" height="230">
 
-## Simple Summary
```

Comment:
Instead of deleting this section please change the title to "Abstract", which follows EIP-1.